idd/Energy+.idd.in

```diff
@@ -23824,6 +23824,40 @@ AirflowNetwork:MultiZone:WindPressureCoefficientValues,
       \units dimensionless
       \note Enter the WPC Value corresponding to the 36th wind direction.
 
+AirflowNetwork:ZoneControl:PressureController,
+	 \memo Define the Pressures control settings for a zone or list of zones.
+	 \memo If you use a ZoneList in the Zone or ZoneList name field then this definition applies
+	 \memo to all the zones in the ZoneList.
+ A1 , \field Name
+      \required-field
+      \reference AirflowNetworkZoneControlPressureControllerNames
+ A2 , \field Zone Name
+      \required-field
+      \type object-list
+      \object-list ZoneAndZoneListNames
+ A3 , \field Control Object Type
+      \required-field
+      \type choice
+      \key AirflowNetwork:MultiZone:Component:ZoneExhaustFan
+      \key AirflowNetwork:Distribution:Component:ReliefAirFlow
+      \note The current selection is AirflowNetwork:MultiZone:Component:ZoneExhaustFan 
+      \note or AirflowNetwork:Distribution:Component:ReliefAirFlow.
+ A4 , \field Control Name
+      \note Control names are names of individual control objects 
+      \required-field
+      \type object-list
+      \object-list ControlTypeNames
+ A5 , \field Pressure Control Availability Schedule Name
+      \note Availability schedule name for pressure controller. Schedule value > 0 means the 
+      \note pressure controller is enabled. If this field is blank, then pressure controller is 
+      \note always enabled.
+      \type object-list
+      \object-list ScheduleNames
+ A6 ; \field Pressure Setpoint Schedule Name
```

Comment:
This needs some \notes to describe what this schedule means, and what the units of the schedule values should be.  Is it common for the pressure setpoint to vary?  It might be nice to have an input for a fixed setpoint rather than require a schedule?